impl/src/test/groovy/com/okta/sdk/impl/resource/log/LogsTest.groovy

```diff
@@ -145,4 +153,50 @@ class LogsTest {
         assertThat log.getRequest().getIpChain().get(0).getGeographicalContext().getGeolocation().lat, equalTo(43.3091d)
         assertThat log.getRequest().getIpChain().get(0).getGeographicalContext().getGeolocation().lon, equalTo(-71.6861d)
     }
-}
\ No newline at end of file
+
+    @Test
+    void testGetLogsBetweenDates() {
+
+        Date sinceDate = Date.from(Instant.from(DateTimeFormatter.ISO_DATE_TIME.parse("2017-11-30T21:15:16.838Z")))
+        Date untilDate = Date.from(Instant.from(DateTimeFormatter.ISO_DATE_TIME.parse("2017-11-30T21:16:00.081Z")))
+        
+        // get log events between dates
+        List<LogEvent> logs = client.getLogs(sinceDate, untilDate, null, null, null).stream().collect(Collectors.toList())
+        logs.forEach { assertThat it, instanceOf(LogEvent) }
+
+        ArgumentCaptor<Request> argument = ArgumentCaptor.forClass(Request.class)
+        verify(client.mockRequestExecutor).executeRequest(argument.capture())
+        assertThat argument.getValue().queryString.since, instanceOf(String)
```

Comment:
Check the specific format of this string to make sure it matches what the Okta API expects

Reply:
I see that date is not handled correctly and will need to fix this issue (created https://oktainc.atlassian.net/browse/OKTA-308568).

Reply:
Fixed in PR - https://github.com/okta/okta-commons-java/pull/36.